### ISSUE TITLE
Broken instalation on Mac

### ISSUE THREAD

**Issue by pevnak** [wsmoses/Enzyme.jl#15]
Hello,

I was curious to test Enzyme to see, if it is faster than my custom written rules. But the installation on Mac is broken
```
julia> autodiff(f1, Active(1.0))
ERROR: could not load library "LLVMEnzyme-9.dylib"
dlopen(LLVMEnzyme-9.dylib, 1): image not found
Stacktrace:
 [1] enzyme! at /Users/tomas.pevny/.julia/packages/Enzyme/QQP8b/src/compiler/optimize.jl:2 [inlined]
 [2] (::Enzyme.Compiler.var"#2#3"{Bool,LLVM.Module,String,LLVM.TargetMachine})(::LLVM.ModulePassManager) at /Users/tomas.pevny/.julia/packages/Enzyme/QQP8b/src/compiler/optimize.jl:82
 [3] LLVM.ModulePassManager(::Enzyme.Compiler.var"#2#3"{Bool,LLVM.Module,String,LLVM.TargetMachine}) at /Users/tomas.pevny/.julia/packages/LLVM/KITdB/src/passmanager.jl:28
 [4] optimize!(::LLVM.Module, ::LLVM.Function; run_enzyme::Bool) at /Users/tomas.pevny/.julia/packages/Enzyme/QQP8b/src/compiler/optimize.jl:13
 [5] Enzyme.LLVMThunk(::Function, ::Tuple{DataType}; optimize::Bool, run_enzyme::Bool) at /Users/tomas.pevny/.julia/packages/Enzyme/QQP8b/src/Enzyme.jl:120
 [6] LLVMThunk at /Users/tomas.pevny/.julia/packages/Enzyme/QQP8b/src/Enzyme.jl:38 [inlined]
 [7] Enzyme.Thunk(::Function, ::Tuple{DataType}) at /Users/tomas.pevny/.julia/packages/Enzyme/QQP8b/src/Enzyme.jl:130
 [8] autodiff(::Function, ::Active{Float64}) at /Users/tomas.pevny/.julia/packages/Enzyme/QQP8b/src/Enzyme.jl:185
 [9] top-level scope at REPL[6]:1
```

Do you have ideas what could go wrong?

**Comment by vchuravy** [comment]
Will be fixed by #14

**Comment by vchuravy** [comment]
@pevnak should be fixed by v0.2.0 if yopu want to give it a try!
